release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -73,6 +73,10 @@ For more information, see xref:../getting_started/openshift-overview.adoc[Gettin
 [id="ocp-4-11-installation-and-upgrade"]
 === Installation and upgrade
 
+[id="ocp-4-11-installation-and-upgrade-hyperv"]
+==== Support for Microsoft Hyper-V generation version 2 images
+By default, the installation program now deploys a Microsoft Azure and a Microsoft Azure Stack Hub cluster using a Hyper-V generation version 2 image. If the installation program detects that the virtual machine cannot support a version 2 image, it falls back to a version 1 image for the deployment.
```

Comment:
I overlooked this but we need to make this a bit more clear. We now support/default virtual machines on Azure with Hyper-V generation version 2. It's more of a VM disk configuration feature rather than an image feature. Terraform controls this by the images deployed to the VMs so they're correlated but let's make it clear it's a VM feature.

```suggestion
By default, the installation program now deploys a Microsoft Azure and a Microsoft Azure Stack Hub cluster using Hyper-V Generation V2 VMs. If the installation program detects that the instance type selected for the virtual machines do not support version 2, it falls back to a version 1 for the deployment.
```

Reply:
Thank you for the clarifications.

Reply:
1. most VM HyperVGenerations ==['V1, V2]', previous we use V1 image, now we use V2 image on the osDisk of the VM instance. 
2. previous version we can not use the VM HyperVGenerations='V2', now we can use it. such as the vm in the standardDCSv3Family, standardDDCSv3Family
3. How about changed to "cluster using Hyper-V Generation V2 images" ?